book-src/01-03-file-modified-24h-ago.md

```diff
@@ -4,6 +4,10 @@ Gets the current working directory by calling `fs.cwd()`, and then iterate files
 
 For each entries, we check if it's a file, and use `statFile()` to retrieve file's metadata.
 
+Note: iterate api has been changed in zig `nightly`, see this [issue](https://github.com/ziglang/zig/pull/18076)!
```

Comment:
Thanks for remind.

If you like, we make this demo compatible with both 0.11.0 and master. Something like:
https://github.com/zigcc/zig-cookbook/blob/8440d5ea1612ca418aa7da488e5ee49610d83cf4/build.zig#L14